Editor/Git/GitTask.cs

```diff
@@ -3,12 +3,12 @@
 
 namespace DUCK.PackageManager.Editor.Git
 {
-	public class GitTask : AbstractProcessTask
+	internal class GitTask : AbstractProcessTask
 	{
 		public static GitTask Run(string args)
 		{
 			var task = new GitTask(args);
-			task.Execute();
+			task.Execute(r => {});
```

Comment:
This was an old util function for testing, it kinda made sense befor but now the API requires `onComplete`.

It's not actually used in anywhere in active code, but I use it a lot during testing. I should remove it from the repo, and jst add it locally and never commit.